tests/rxjava/src/test/kotlin/RxJavaTest.kt

```diff
@@ -63,18 +57,5 @@ class RxJavaTest {
           it is ApolloException
         }
   }
-
-  @Test
-  fun writingToTheStoreWorks() {
-    rx2ApolloStore.rxWriteOperation(GetRandomQuery(), GetRandomQuery.Data(random = 43)).blockingGet()
```

Comment:
This is the part that I feel is better not shipping until we get a better idea of the problem we're trying to solve.

Note how the rest of the tests didn't change